geoutils/geoviewer.py

```diff
@@ -1,98 +1,170 @@
 
+import argparse
 import sys
+from typing import Any
 
-import argparse
-import numpy as np
 import matplotlib.pyplot as plt
+import numpy as np
 
 from geoutils.georaster import Raster
-from geoutils.geovector import Vector
 
 
-def getparser():
+def getparser() -> argparse.Namespace:
 
     # Set up description
-    parser = argparse.ArgumentParser(description='Visualisation tool for any image supported by GDAL.')
-
-    #Positional arguments
-    parser.add_argument('filename', type=str, help='str, path to the image')
-
-    #optional arguments
-    parser.add_argument('-cmap', dest='cmap', type=str, default='default', help='str, a matplotlib colormap string (default is from rcParams).')
-    parser.add_argument('-vmin', dest='vmin', type=str, default=None, help='float, the minimum value for colorscale, or can be expressed as a percentile e.g. 5%% (default is calculated min value).')
-    parser.add_argument('-vmax', dest='vmax', type=str, default=None, help='float, the maximum value for colorscale, or can be expressed as a percentile e.g. 95%% (default is calculated max value).')
-    parser.add_argument('-band', dest='band', type=int, default=None, help='int, which band to display (start at 0) for multiband images (Default is 0).')
-    parser.add_argument('-nocb', dest='nocb', help='If set, will not display a colorbar (Default is to display the colorbar).',action='store_false')
-    parser.add_argument('-clabel', dest='clabel', type=str, default='', help='str, the label for the colorscale (Default is empty).')
-    parser.add_argument('-title', dest='title', type=str, default='', help='str, figure title (Default is empty).')
-    parser.add_argument('-figsize', dest='figsize', type=str, default='default', help='str, figure size, must be a tuple of size 2, either written with quotes, or two numbers seperated by coma, no space (Default is from rcParams).')
-    parser.add_argument('-max_size', dest='max_size', type=int, default=2000, help='int, image size is limited to max_size**2 for faster reading/displaying (Default is 2000).')
-    parser.add_argument('-save', dest='save', type=str, default='', help='str, filename to the output filename to save to disk (Default is displayed on screen).')
-    parser.add_argument('-dpi', dest='dpi', type=str, default='default', help='int, dpi value to use when saving figure (Default is from rcParams).')
-    parser.add_argument('-nodata', dest='nodata', type=str, default='default', help='float, no data value (Default is read from file metadata).')
-    parser.add_argument('-noresampl', dest='noresampl', default=False, action='store_true', help='True or False, if False then allow dynamic image downscaling, if True, prevent it.')
+    parser = argparse.ArgumentParser(description="Visualisation tool for any image supported by GDAL.")
+
+    # Positional arguments
+    parser.add_argument("filename", type=str, help="str, path to the image")
+
+    # optional arguments
+    parser.add_argument(
```

Comment:
Why are so many spaces and breaklines added here? It looks a bit awful now.

Reply:
`black` is subtitled "The Uncompromising Code Formatter" for a reason ;) The lengths help texts drive the expressions to not fit in one-liners, so `black` divides them accordingly.

Reply:
Ok, sometimes I forget that the GitHub interface adds empty lines in the diffs. so there are actually no blank lines.
But I still don't understand why black does not add a line break when the line is close to 120 characters, rather than after each argument...